feat:(promotion page): A topic to describe product promotions and enrollment (#20204)
newrelic/docs-website: src/content/docs/accounts/accounts/account-maintenance/promotions-ui.mdx

---
title: Product promotions
metaDescription: "How to sign up for product promotions"
tags:
  - Accounts
  - Accounts and billing
  - General account settings

freshnessValidatedDate: never 
---

> Promotions are special offers to try out New Relic product features. You can browse through our various promotions and manage your enrollment in the UI at <DNT>**Previews & trials**</DNT>. 

Reviewer: ```suggestion
Promotions are special offers to try out New Relic product features. You can browse through our promotions and manage your enrolment on the <DNT>**Previews & trials**</DNT> page. 
```

The current available promotion is product previews, which lets you try a feature before its official release. From the **<DNT>Previews & trials</DNT>** page you can get:

* **Centralized access**: View all available product previews in one location.
* **Enrollment flexibility**: Join or leave product previews until they’re generally available (GA).
* **Upgrade options**: Upgrade to access products when they reach GA.

## Eligibility [#eligibility]

> To check the available product previews for your organization, navigate to <DNT>**[one.newrelic.com > Administration](https://one.newrelic.com/admin-portal) > Preview & Trials**</DNT>. The list of available product previews appears. If you do not have a product preview you want to enroll in, your organization may not be eligible for that. Reach out to your billing admin to enroll your organization in the product preview.

Reviewer: ```suggestion
To check the available product previews for your organization, navigate to <DNT>**[one.newrelic.com > Administration](https://one.newrelic.com/admin-portal) > Preview & Trials**</DNT>. If you do not see the product preview for which you want to enroll, your organization may not be eligible for that. Reach out to your billing admin to enroll your organization in the product preview.
```

### Eligible users [#users]

Product previews are available to the entire organization, covering all users and accounts within it. The enrollment options may vary depending on the product preview. Some previews allow any user within the organization to enroll, while others limit enrollment, leaving, or re-enrollment to billing users only.

### Eligible organizations [#org-rules]

Organizations can enroll in multiple product previews concurrently. Eligibility for a preview depends on various factors. If you get an error message while trying to opt into a promotion, reach out to your billing admin. You may need to perform additional steps for enrollment, or your organization may not be eligible for the preview.

## Manage your enrollment [#access-ui]

> The <DNT>**Previews & trials**</DNT> page lists available product preview in a single, consolidated table view. Each row in the table contains essential information such as the product name, a brief description, status (<DNT>**Available**, **Opted In**, **Opted Out**</DNT>), and the anticipated end date of the preview.

Reviewer: Is the second sentence necessary?
